explore/src/main/scala/explore/programs/ProgramDetailsTile.scala

```diff
@@ -41,19 +42,25 @@ object ProgramDetailsTile:
     useContext(AppContext.ctx).map: ctx =>
       import ctx.given
 
-      val details: ProgramDetails        = props.programDetails.get
-      val thesis: Boolean                = details.allUsers.exists(_.thesis.exists(_ === true))
-      val users: View[List[ProgramUser]] = props.programDetails.zoom(ProgramDetails.allUsers)
-      val newDataNotificationView        =
+      val details: ProgramDetails               = props.programDetails.get
+      val thesis: Boolean                       = details.allUsers.exists(_.thesis.exists(_ === true))
+      val users: View[List[ProgramUser]]        = props.programDetails.zoom(ProgramDetails.allUsers)
+      val newDataNotificationView               =
         props.programDetails
           .zoom(ProgramDetails.shouldNotify)
           .withOnMod(b => ProgramQueries.updateGoaShouldNotify[IO](props.programId, b).runAsync)
+      val cfpActivePeriod: Option[DateInterval] = details.proposal.flatMap(_.call).map(_.active)
+
+      // We `should` always have a call for proposal if we get here, but...
+      def dateOrMissing(o: Option[LocalDate], label: String) =
```

Comment:
Maybe we can just omit the element if the date is missing?

Also, could we add explicit types?

Reply:
We could do that, but I thought it might be easier to debug in the future if we ever run into this again. It `should` never happen since the program tab is only for ACCEPTED proposals, which should have a CfP. But, omitting them would be cleaner, I guess.